contracts/receivers/aave-v2-receiver/interfaces.sol

```diff
@@ -5,12 +5,12 @@ interface AccountInterface {
     function enable(address) external;
     function disable(address) external;
     function isAuth(address) external view returns (bool);
-    function cast(
+    function castMigrate(
```

Comment:
@KaymasJain Here we are going to use `cast`, right?

Reply:
Yeah! We can use the normal `cast`. There shouldn't be any issue. Right?

Reply:
Or else we can change it to `castMigrate` with some restriction.